tools/templates/cc.yml

```diff
@@ -33,6 +33,13 @@ jobs:
       projects: '**/tests/*.csproj'
       arguments: '--configuration Release /p:CollectCoverage=true /p:CoverletOutputFormat=cobertura'
       publishTestResults: false
+  - script: |
+      bash <(curl -s https://codecov.io/bash)
```

Comment:
I am a bit concerned about this - these are IMO top security issues since they allow arbitrary code execution on our build servers and potentially secret exfiltration.   Need a better way.

Reply:
right, better not use it here